semgrep/tests/e2e/test_ci.py

```diff
@@ -92,14 +89,15 @@ def git_tmp_path_with_commit(monkeypatch, tmp_path):
         ["git", "rev-parse", "HEAD"], encoding="utf-8"
     ).strip()
 
-    yield (repo_base, base_commit, head_commit)
-
+    mocker.patch.object(
+        GitlabMeta, "_fetch_branch_get_merge_base", return_value=base_commit
+    )
 
-import contextlib
+    yield (repo_base, base_commit, head_commit)
 
 
-@contextlib.contextmanager
-def ci_mocks(base_commit, autofix):
+@pytest.fixture(autouse=True)
+def automocks(mocker):
```

Comment:
I made this an autouse fixture, so instead of using it as a context manager, these mocks get applied to every test case in this file automatically.